CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.3)
+project(function)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+
+set(SOURCE_FILES assignment3.cpp read_config.h)
+# add_executable(function ${SOURCE_FILES})
+add_library (<read_config> ${SOURCE_FILES})
```

Comment:
we don't want the < and > around the property because the projects name is `function`